rollbar/__init__.py

```diff
@@ -445,6 +472,12 @@ def search_items(title, return_fields=None, access_token=None, endpoint=None, **
                     **search_fields)
 
 
+def wait(f=None):
+    _threads.join()
+    if f is not None:
+        return f()
```

Comment:
This allows wait to return a value which we use to return the value at the end of the lambda function